test/statsCases/context-independence/webpack.config.js

```diff
@@ -26,7 +26,7 @@ const base = (name, devtool) => ({
 	entry: {
 		main: {
 			import: "./index",
-			layer: path.resolve(__dirname, name)
+			layer: "my-layer"
```

Comment:
Changed because we can't be context independence here, we use different layers, so we have different modules